tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/interpreter/DebugInterpreter.java

```diff
@@ -63,6 +63,15 @@ static void logOnDeviceObject(Object object, TornadoXPUDevice interpreterDevice,
         appendLogBuilder(verbose, logBuilder);
     }
 
+    static void logPersistedObject(Object object, TornadoXPUDevice interpreterDevice, StringBuilder logBuilder) {
+        String verbose = String.format("bc: %s[0x%x] %s on %s", //
+                InterpreterUtilities.debugHighLightBC("PERSIST_BUFFER"), //
```

Comment:
I would prefer just `PERSIST` instead of `PERSIST_BUFFER`. It is aligned with the name of the bytecode in `TornadoVMBytecodes.java`.